Copy documentation from Kyma to the module's repo (#301)
kyma-project/istio: docs/user/01-50-default-istio-setup.md

---
title: Default Istio setup in Kyma
---

> Istio in Kyma is installed with the help of the `istioctl` tool. The tool is driven by a configuration file containing an instance of the [IstioOperator](https://istio.io/docs/reference/config/istio.operator.v1alpha1/) custom resource.

Reviewer: ```suggestion
Istio in Kyma is installed with the help of the [Istio library](https://github.com/istio/istio/tree/master/operator).
```

## Istio components 
<!-- we have the tech design doc (that has to be updated) -->

> This list shows the available Istio components and addons. Check which of those are enabled in Kyma:

Reviewer: ```suggestion
This list shows the available Istio components and addons. Check which of those are enabled with Istio module:
```

- Istiod (Pilot)
- Ingress Gateway

> - Grafana - installed as separate component - [monitoring](../../05-technical-reference/00-architecture/obsv-01-architecture-monitoring.md)
> - Prometheus - installed as separate component - [monitoring](../../05-technical-reference/00-architecture/obsv-01-architecture-monitoring.md)

Reviewer: ```suggestion
```
